src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -273,6 +273,18 @@ def load_pytorch_state_dict_in_tf2_model(
             new_key = key.replace("running_var", "moving_variance")
         if "running_mean" in key:
             new_key = key.replace("running_mean", "moving_mean")
+
+        # New `weight_norm` from https://github.com/huggingface/transformers/pull/24030
+        key_components = key.split(".")
+        _name = None
```

Comment:
Let's use another name if `name` is taken, but no leading underscore in variable names please.